File: 2.0/java/texts/createTextAutoReply.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.TextAutoReply;
import com.callfire.api.client.api.common.model.ResourceId;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        TextAutoReply textAutoReply = new TextAutoReply();
        textAutoReply.setNumber("67076");
        textAutoReply.setMessage("I am a leaf on the wind");
        textAutoReply.setKeyword("CALLFIRE");
        ResourceId resourceId = client.textAutoRepliesApi().create(textAutoReply);
    }
}

File: 2.0/csharp/texts/createTextAutoReply.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var textAutoReply = new TextAutoReply()
        {
            Number = "67076",
            Message = "I am a leaf on the wind",
            Keyword = "CALLFIRE"
        };
        ResourceId resourceId = client.TextAutoRepliesApi.Create(textAutoReply);
    }
}
